SETUP.md

```diff
@@ -93,23 +101,61 @@ openssl base64 -A -in deploy.key > deploy.key.txt
 echo "https://github.com/$OWNER/$REPO/settings/keys/new"
 
 # Print the public key for copy-pasting to GitHub
-cat deploy.key.pub
+cat ci/deploy.key.pub
 ```
 
 Go to the GitHub settings URL echoed above in a browser, and click "Add deploy key".
 For "Title", add a description like "Manubot Travis Deploy Key".
-Copy-paste the contents of the `deploy.key.pub` text file (printed above by `cat`) into the "Key" text box.
+Copy-paste the contents of the `ci/deploy.key.pub` text file (printed above by `cat`) into the "Key" text box.
 Check the "Allow write access" box below.
 Finally, click "Add key".
 
-#### Add the private key to Travis CI
+#### Add the private key to GitHub
+
+```sh
+# Print the URL for adding the private key to GitHub
+echo "https://github.com/$OWNER/$REPO/settings/secrets"
+
+# Print the encoded private key for copy-pasting to Travis CI
+cat ci/deploy.key.txt && echo
+```
+
+Next, go to the GitHub repository settings page (URL echoed above).
+Click "Add a new secret".
+For "Name", enter `MANUBOT_SSH_PRIVATE_KEY`.
+Next, copy-paste the content of `deploy.key.txt` into "Value"
+(printed above by `cat`, including any trailing `=` characters if present).
+
+### GitHub Actions
+
+If you plan on only using GitHub actions, you can remove configuration files for other CI services:
```

Comment:
I think this is fine for now instead of removing the files from rootstock.  Some users may prefer to continue using Travis CI for reasons we can't anticipate.